src/autolua.zig

```diff
@@ -172,6 +172,11 @@ pub fn check(L: ?*lua.lua_State, idx: c_int, comptime T: type) T {
             // lua.lua_pop(L, 1);
             // const ptr = lua.lua_touserdata(L, idx);
         },
+        .Optional => |N| {
+            if (lua.lua_type(L, idx)<lua.LUA_TBOOLEAN)
```

Comment:
```suggestion
            if (lua.lua_type(L, idx) < lua.LUA_TBOOLEAN)
```
This is more consistent with the rest of the code.

Reply:
that's right, wait imma fix it rn